server/server.go

```diff
@@ -70,7 +78,8 @@ func upload(w http.ResponseWriter, r *http.Request) {
 	w.Write(response)
 }
 
-// Builds the assignment uploaded.
+// build compiles the uploaded assignment by using the command to compile
+// It sends the compilation status or message as a response to the requester.
```

Comment:
remove "to the requester" and maybe replace with "to the client" or nothing.

Reply:
Okay